modules/ztp-creating-the-policygentemplates.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-creating-the-policygentemplates_{context}"]
+= Creating the PolicyGenTemplates
+
```

Comment:
Based on the module templates, please add an introductory sentence or paragraph for the procedure.

Reply:
Added - "Use the following procedure to create the `PolicyGenTemplates` you will need for generating policies in your 
Git repository for the hub cluster.